labchecker/plugins/nodepool/network.py

```diff
@@ -0,0 +1,32 @@
+from labcheck import Plugin
+from plugins.recover import Recover
+import commands
+
+
+class NetworkPlugin(Plugin):
+    ptype = 'nodepool'
+    name = 'network'
+
+    def check(self):
+        self.failed = False
+        self.reasons = []
+        net = 'openstack --os-cloud %s network list -f value -c Name | grep openlab-net' % self.cloud
```

Comment:
1. Why do not use `network show openlab-net`? it's faster
2. If there are more than one `openlab-net`, it's wrong cloud setting too.

Reply:
1. Yes, I will update in next PS
2. ++